src/components/AccountContent/Review.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+const Review = () => {
```

Comment:
Since we are using the same name as mentioned [here](https://github.com/Cobda/cobda-web/pull/24#discussion_r576278004), Should we go with `ProductReview` instead?